app/models/user.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable require-atomic-updates */
+const bcrypt = require('bcrypt');
+
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'users',
+    {
+      name: {
+        type: DataTypes.STRING,
+        allowNull: false,
+        validate: {
+          notNull: {
+            args: true,
+            msg: 'Name cannot be null'
+          },
+          notEmpty: {
+            args: true,
+            msg: 'Name cannot be empty'
+          }
+        }
+      },
+      last_name: { type: DataTypes.STRING },
+      email: {
+        type: DataTypes.STRING,
+        validate: {
+          isEmail: true
+        },
+        unique: {
+          args: true,
+          msg: 'Email address already in use!'
+        }
+      },
+      password: {
+        type: DataTypes.STRING,
+        validate: {
+          len: {
+            args: [8, 60],
+            msg: 'Password length should be between 8 and 60'
+          }
+        }
+      }
+    },
+    {
+      timestamps: false,
```

Comment:
you should mark this option as `true`. most probably you need to know when a user was created or modified.

Reply:
done :)